tests/diagnostics/test_windspeed.py

```diff
@@ -0,0 +1,30 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES.
+# SPDX-FileCopyrightText: All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pytest
+import torch
+from earth2mip.schema import Grid
+from earth2mip.diagnostic import WindSpeed
+
+
+@pytest.mark.parametrize("device", ["cuda:0"])
+@pytest.mark.parametrize("grid", [Grid.grid_721x1440, Grid.grid_720x1440])
+def test_wind_speed(device, grid):
+    model = WindSpeed.load_diagnostic(None, level="10m", grid=grid)
```

Comment:
Let's avoid `load_diagnostic`

```suggestion
    model = WindSpeed(level="10m", grid=grid)
```

Reply:
Keeping this one since this is part of the diagnostics presently and all others load the same way. If it changes then the tests need to get updated.